tools/compileExtensions.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# vim: set fileencoding=utf-8 :
+
+# make.py
+# An Arma 3 addon build system
+
+###############################################################################
+
+# The MIT License (MIT)
+
+# Copyright (c) 2013-2014 Ryan Schultz
```

Comment:
? :D
I don't think this header is correct haha. I added Extension building to make.py anyways, so you can get rid of the entire header.